inspire/base/templates/format/record/Inspire_Default_HTML_brief.tpl

```diff
@@ -30,21 +30,25 @@
       <div class="col-md-9"  id="left-column">
         <h4 class="custom-h">
           <b>
+            {% set title_list = [] %}
             {% if record.titles %}
               {% for title in record['titles'] %}
+                {% do title_list.append(title['title']) %}
```

Comment:
Yes it is possible! :) I updated the PR.

2015-10-20 15:31 GMT+02:00 Javier Martin Montull <notifications@github.com>:

> In inspire/base/templates/format/record/Inspire_Default_HTML_brief.tpl
> <https://github.com/inspirehep/inspire-next/pull/429#discussion_r42494754>
> :
>
> >              {% if record.titles %}
> >                {% for title in record['titles'] %}
> > +                {% do title_list.append(title['title']) %}
>
> I am not sure if we are doing too much here? Now with an stable data model
> record['titles'] should always be a list.
>
> Can you check if record['titles[0].title'] works? Note that this special
> notation is possible because the record object is a SmartDict from
> Invenio.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/inspirehep/inspire-next/pull/429/files#r42494754>.
>